gen_cert.py

```diff
@@ -137,7 +147,7 @@ def interstitial_factory():
         try:
             self.long_org = long_org or cert_data.get('LONG_ORG', '').encode('utf-8') or settings.DEFAULT_ORG
             self.long_course = cert_data.get('LONG_COURSE', '').encode('utf-8') or long_course or ''
-            self.issued_date = issued_date or cert_data.get('ISSUED_DATE', '').encode('utf-8')
+            self.issued_date = issued_date or cert_data_issued_date
```

Comment:
Later on, `self.issued_date` is used to render the verification page.
Currently, it expects a `YYYY-MM-DD` formatted string.